lib/project.rb

```diff
@@ -34,24 +35,30 @@ def diff_checker(diff)
     exit(0)
   end
 
-  def init_multilog(debug)
-    log = File.open("#{workspace_path}/#{tag}.log", 'a')
-    log.sync = true
-    @logger = MonoLogger.new MultiDelegator.delegate(:write, :close)\
-                                           .to(STDOUT, log)
-    @logger.datetime_format = '%Y-%m-%d %H:%M:%S'
+  def pre_init_multilog(debug)
+    @temp_pre_logger_file = Tempfile.new('')
+    @temp_pre_logger_file.sync = true
+    @pre_logger = MonoLogger.new(@temp_pre_logger_file)
+    @stdout_logger = MonoLogger.new(STDOUT)
+    @logger = MultiLogger.new(@pre_logger, @stdout_logger)
     @logger.level = debug ? 'DEBUG' : 'INFO'
   end
 
+  def init_multilog
```

Comment:
Better change to "save previous log" or something similar.

Reply:
Changed